db/migrate/20171106074703_add_random_token_to_user.rb

```diff
@@ -0,0 +1,9 @@
+class AddRandomTokenToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :token, :string
+    User.all.each do |user|
```

Comment:
I don't recommend altering data in your db migrations. Database migrations should only be to set up your schema. For any data related changes, add that to your seed file or a separate rake task. 
Unless you plan to have a few users when someone first runs your application, then I would leave this data update out entirely. Probably best to run this from the console.